pkg/io/storage/storage_test.go

```diff
@@ -0,0 +1,57 @@
+package storage
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestFileVault_SaveData(t *testing.T) {
+
+	fileVault, err := NewFileVault("file")
+	defer func() {
+		err := fileVault.Close()
+		if err != nil {
+			t.Error("file not closed")
+		}
+	}()
+	if err != nil {
+		fmt.Println(err)
+		t.Error("fileVault = nil")
+	}
+	got := []byte("Hey")
+
+	err = fileVault.SaveData([]byte("f1"), got)
+	if err != nil {
+		t.Error("file not found or something wrong")
+	}
+
+	wont := fileVault.storage["f1"]
+	if string(wont) != string(got) || err != nil {
+		t.Error("file not found or something wrong")
+	}
+}
+
+func TestFileVault_ReadData(t *testing.T) {
+
+	fileVault, err := NewFileVault("file")
+
+	defer func() {
+		err := fileVault.Close()
+		if err != nil {
+			t.Error("file not closed")
+		}
+	}()
+
+	if err != nil {
+		fmt.Println(err)
+		t.Error("fileVault = nil")
+	}
+	wont := []byte("World")
+	fileVault.storage["f2"] = wont
```

Comment:
That not quite a check, because by your logic you have to have `f2` value in your file, not in the storage.